hvplot/converter.py

```diff
@@ -398,7 +402,8 @@ def __init__(self, data, x, y, kind=None, by=None, use_index=True,
                    'padding', 'xformatter', 'yformatter',
                    'height', 'width', 'frame_height', 'frame_width',
                    'min_width', 'min_height', 'max_width', 'max_height',
-                   'fontsize', 'responsive', 'shared_axes', 'aspect', 'data_aspect']
+                   'fontsize', 'fontscale', 'responsive', 'shared_axes', 
```

Comment:
so i added fontscale here to variable plotwds. Is this correct? When should keywords be added to plotwds?

Reply:
There was some refactoring done here, which I unfortunately did not follow completely. I **think** only kwds that are not listed explicitly should be added and only kwds that need some further processing should be listed explicitly, so maybe drop fontsize from the `__init__` as part of this PR as well.